mod/fbrowser.php

```diff
@@ -10,6 +10,7 @@
 /**
  * @param App $a
  */
+/// @TODO & is missing or App ?
```

Comment:
As I wrote in my comments (TODO) in source code, the & allows this:

```
function foo (App &$a) {
$a = new App();
}
```
And it has been changed for all other usages and will become harder to track where it is has been changed. Simply remove the `&` and $a can only be changed locally (sadly as PHP does not support `final` in parameter list) but not globally (very where).